src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,54 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  setMovies: React.Dispatch<React.SetStateAction<Movie[]>>;
+};
+
+export const FindMovie: React.FC<Props> = ({ setMovies }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [inputAlert, setInputAlert] = useState(false);
+  const [inputValue, setInputValue] = useState('');
+
+  const requestFilm = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    e.preventDefault();
+    getMovie(inputValue).then(res => {
+      if ('Title' in res) {
+        const poster = res.Poster === 'N/A'
+          ? 'https://via.placeholder.com/360x270.png?text=no%20preview'
```

Comment:
please move ''https://via.placeholder.com/360x270.png?text=no%20preview'' to the constant